lib/README.md

```diff
@@ -0,0 +1,19 @@
+# TeselaGen Python Tools.
+
+This package includes some Python tools to use in combination with TeselaGen platform.
+These tools includes _TeselaGen Python API Client_ 
+
+This package runs on Python 3.6 and above.
+
+**NOTE :** All the following commands are supposed to be run on the `base` directory, unless specified.
+
+## Library Installation
+This library contains the TeselaGen Python API Client.
+
+To install it locally,
+
+1. By using PIP
+
+    Use `pip install teselagen`
+
+
```

Comment:
is this a duplicate of the following  README file ?

*  https://github.com/TeselaGen/api-client/blob/ebe8d6e1831ee8cb23a1be9c7900dc7825a40b16/README.md

Reply:
The `/lib/README.md` is uploaded to PyPI an the other don't. I think is ok to maintain different readmes